lib/issue_exporter/cli.rb

```diff
@@ -26,7 +26,7 @@ def run
       end
 
       fail UsageError, 'missing argument' if ARGV.empty?
-      fail UsageError, 'incorrect number of arguments' if ARGV.count != 3
+      fail UsageError, 'incorrect number of arguments' unless correct_number_of_args ARGV.count
```

Comment:
Line is too long. [95/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.